### DISCUSSION TITLE
Bootstrap Talos on Hetzner Bare Metal Servers

### DISCUSSION THREAD

**Discussion by mrkhachaturov** [syself/cluster-api-provider-hetzner#1482] [Q&A] [answered]
Hello! 

Is it possible to install Talos on Hetzner Bare Metal Servers using CAPH?

`clusterctl init --bootstrap talos --control-plane talos --infrastructure hetzner ` 

Hetzner vSwitch is supported in CAPH?

**Comment by Preisschild** [comment, marked as answer]
No.

See https://github.com/syself/cluster-api-provider-hetzner/issues/133
